### DISCUSSION TITLE
Python Docs Help!! (please)

### DISCUSSION THREAD

**Discussion by PhantomLel** [mjansen4857/pathplanner#590] [Q&A]
Hi, my team is very new to using python for the FRC robot and we are struggling to figure how to implement PathPlanner for autonomous. We are using a swerve drive system and despite looking at the docs for python we are unable to figure out how to use anything. This project looks really good but I just need some example on how to get started with it using RobotPy. 

Thanks in advanced!

**Comment by Pythonrobotics12** [comment]
I also have this problem, please help!

**Reply by PhantomLel** [reply]
I managed to figure out some of the structure but I'm struggling to figure out what is wrong here. If someone could please take a look at my code and see if anything is glaringly wrong it would be greatly appreciated.

https://github.com/OlympusRobotics/Olympus_Robotics_FRC/tree/main/perry

The code calls the command in auto init but it just doesn't call the drive function at all and gives and loop overflow error.
